contrib/charts/cert-manager/values.yaml

```diff
@@ -6,29 +6,41 @@ replicaCount: 1
 image:
   repository: quay.io/jetstack/cert-manager-controller
   tag: v0.2.2
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
 
 createCustomResource: true
 
 rbac:
-  enabled: true
+  create: true
+  serviceAccountName: default
+
+# Optional additional arguments
+extraArgs: []
+  # - --cluster-resource-namespace=ingress
 
 resources:
   requests:
     cpu: 10m
     memory: 32Mi
 
+nodeSelector: {}
```

Comment:
This isn't used in the actual templates from what I can see

Reply:
It is. I was actually there before. that's why you don't see it in the diff. I just added the empty default to `values.yaml`.